docs/020-deployment/howtos/080-configure-dns.md

```diff
@@ -108,14 +115,10 @@ If you have any questions about DNS configuration, you can request CluedIn suppo
           - '*.mycompany.com'
     ```
 
-1. If you are using multiple environments (development, test, and production), update the TLS hosts section for each environment in the `values.yaml` file:
-   - Development – replace all instances of `20.0.189.11.sslip.io` to `dev.mycompany.com`
-   - Test – replace all instances of `20.0.189.12.sslip.io` to `test.mycompany.com`
-   - Production – replace all instances of `20.0.189.13.sslip.io` to `prod.mycompany.com`
 1. Save the file and apply changes from the local file to the CluedIn cluster by running the following command:  
-    `helm upgrade -i cluedin-platform cluedin/cluedin-platform  -n cluedin --create-namespace  --values Cluster-Current-values.yaml`
+    `helm upgrade -i cluedin-platform cluedin/cluedin-platform -n cluedin --values Cluster-Current-values.yaml`
 
 After a short time, you'll see the confirmation of your update in the console. CluedIn is now configured to use your new DNS address.  
 ![configure-dns-2.png](../../assets/images/ama/howtos/configure-dns-2.png)
 
-**Note:** This will use the LetsEncrypt service in the cluster to do an HTTP request to validate the certificate. If you would like to use a self-provided certificate, please review the Configure [`TLS Certificates`](/deployment/infra-how-tos/configure-certificates) page
\ No newline at end of file
+**Note:** This will use the LetsEncrypt service in the cluster to do an HTTP request to validate the certificate. If you would like to use a self-provided certificate, please review the [`Configure TLS Certificates`](/deployment/infra-how-tos/configure-certificates) page
```

Comment:
```suggestion
**Note:** This will use the LetsEncrypt service in the cluster to do an HTTP request to validate the certificate. If you would like to use a self-provided certificate, please review the [Configure TLS Certificates](/deployment/infra-how-tos/configure-certificates) page.
```